src/Cli/dotnet/Commands/Test/MTP/Terminal/HumanReadableDurationFormatter.cs

```diff
@@ -16,6 +16,7 @@ public static void Append(ITerminal terminal, TimeSpan duration, bool wrapInPare
             terminal.Append('(');
         }
 
+        // TODO: Do these abbrevations (d for days, h for hours, etc) need to be localized?
```

Comment:
Ping @baronfel

Reply:
I should have opened an issue to follow-up.